react/react_intro/README.md

```diff
@@ -28,41 +28,53 @@
+React is a JavaScript library that was created by Facebook. Developers use React to build out the UI and frontend logic for web applications. Up until now, we have been building our UI by writing a single large HTML file, then linking it to a JavaScript file. The JavaScript file uses [DOM manipulation](https://github.com/joinpursuit/Pursuit-Core-Web/tree/master/html_css_dom/dom_1) to edit, add, and remove elements in response to user interaction. This approach has worked well for us so far, but has some challenges when scaling to larger applications. Consider the following image:
 
 ![Facebook homepage redesign](https://cdn.vox-cdn.com/thumbor/0KnV_DIxDm00kPsX8hR0f4ZzMIU=/0x0:2048x1410/920x613/filters:focal(861x542:1187x868):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/63696407/facebook_website_redesign_1.9.jpg)
 
 
-There are at least four entirely distant components that are on the page including:
+There are at least four entirely distant UI elements on the page, including:
 
 - the top bar
 - the left sidebar
-- the stories feed
-- the contacts list.  
+- the stories and feed section
+- the contacts list
 
-Using our DOM manipulation approach, we would have all of these different elements in a single html file.  This would be challenging to maintain because we a bug in one area could easily affect the rest of the page.  
+These are illustrated here:
 
-React solves this problem by introducing `Components`.  Instead of laying out all of our HTML in a single file, we can separate it into separate classes and combine those classes together.
+![Facebook redesign separated into components](./images/fbInterface.jpg)
+
+Using our DOM manipulation approach, we would have all of these different elements in a single HTML file. This would be challenging to maintain because a bug in one area could easily affect the rest of the page. Making matters worse, the bug would be difficult to find. A developer would have to sift through a single huge file to find the issue - think finding a needle in a haystack.
+
+React solves this problem by introducing `Components`. Instead of laying out all of our HTML and JS in single files, we can separate them into separate files based on what part of the UI they are responsible for rendering. We then import these files, organize them, and let React combine their markup and functionality into a single, cohesive page. If we have an issue in one component, we simply open that component's file, and the code that we need to fix is right there.
+
+Remember, however, that a web browser can't read and combine components by itself. Even if a developer would like neatly separated files, the browser still needs a single HTML page. The aim of React is to improve quality-of-life for developers while packaging frontend applications in a format that browsers can still read and render efficiently.
```

Comment:
I have some thoughts on this section. I don't think the focus on single files vs multiple files is the best possible explanation for why React was created or why React is important to know: https://github.com/joinpursuit/Pursuit-Core-Web/pull/132#discussion_r584959834

Reply:
Right. I think I touch on some of this in the third paragraph, but I can edit the first two so that it doesn't feel like React is mostly good for file organization.